Changelogs for Digital Twin Manager Microservice 1025.1.0 (#4572)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-1-0-improve-performance-of-unlink-asset-from-device-by-batching-operations-per-device-ctm-2861-2865.md

---

> date: ""

Reviewer: ```suggestion
date: "2026-04-21"
```

> title: "Improve performance of unlinkAssetFromDevice by batching operations per device. [CTM-2861] (#2865)"

Reviewer: ```suggestion
title: "Improved performance of unlinkAssetFromDevice by batching operations per device"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2861"
version: "1025.1.0"
---

> Optimized unlinkAssetFromDevice(...) by grouping operations per device,

Reviewer: ```suggestion
Optimized unlinkAssetFromDevice endpoint of Assets API by grouping operations per device,
```

> Optimized unlinkAssetFromDevice endpoint of Assets API by grouping operations per device,

Reviewer 2: ```suggestion
To improve the performance, the `unlinkAssetFromDevice` endpoint of the Assets API has been optimized by grouping operations per device,
```
